packit_service/models.py

```diff
@@ -348,11 +336,6 @@ def get_srpm_builds(self):
     def get_test_runs(self):
         return self._get_run_item(model_type=TFTTestRunTargetModel)
 
-    @property
-    def commit_sha(self) -> str:
-        project_event = self.get_project_event()
-        return project_event.commit_sha if project_event else None
-
```

Comment:
I thought we need this one, but looking at the tests it seems we don't.
Thanks for the fix!